contracts/CDPosition.sol

```diff
@@ -38,6 +39,8 @@ contract CDPosition {
         _;
     }
 
+    constructor(address admin) AccessController(admin) {}
```

Comment:
Maybe remove this lint error "Code contains empty blocks"

Reply:
Yeah I'd like to override it for constructors and init functions. For other functions I think the warning is useful. It's a warning so it won't error out in tests. I'm going to do a small investigation into extending the lint rules as I think this will be useful for enforcing other things as well. Let's leave this for now in this PR